src/libraries/Microsoft.PowerFx.Interpreter/PrettyPrint.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using Microsoft.PowerFx.Core.IR;
+using Microsoft.PowerFx.Core.Parser;
+using Microsoft.PowerFx.Syntax;
+
+namespace Microsoft.PowerFx.Interpreter

```

Comment:
>Interpreter [](http://example.com/codeflow?start=28&length=11)

Should this be in the interpreter or in the core? We don't need the interpreter to parse an expression (it can be done entirely in core)

Reply:
I don't think we need this in core. I need it to be used in NL2FX and I used it for our final post processing expression so I want input is expression

Reply:
This should be in Core.  But probably on an existing class. I can look around...

(NL2Fx doesn't use interpreter. )

Reply:
actually, it looks like TexlNode.ToString() already does this. 
So we don't need this function.

Reply:
Thanks to point out. abandon this PR since don't need.